build.sbt

```diff
@@ -48,7 +48,11 @@ lazy val examples = project
     Compile / run / fork := true
   )
 
+import Site.SiteConfig
 lazy val site = project
   .in(file("site"))
   .enablePlugins(TypelevelSitePlugin)
   .dependsOn(core.jvm)
+  .settings(
+    laikaConfig := SiteConfig.laikaConfig
```

Comment:
We totally can, but I segregated the `site` configurations in case we wanted to tune a bit more the site (with custom colors, for example) in the future, so we didn't enlarge the `build.sbt` a lot. Should I segregate them or merge the config anyways and worry about that matter in the future?